pandas/tests/tseries/frequencies/test_inference.py

```diff
@@ -449,7 +449,7 @@ def test_series_period_index(freq):
         frequencies.infer_freq(s)
 
 
-@pytest.mark.parametrize("freq", ["M", "L", "S"])
+@pytest.mark.parametrize("freq", ["ME", "L", "S"])
```

Comment:
wasnt L deprecated?

Reply:
yup, but looks like it's only in `to_timedelta`: https://github.com/pandas-dev/pandas/pull/53557

should they be deprecated completely?

Reply:
i lean that way yes